### PR TITLE
Added head_cam frame to urdf

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by NadiaHG** [comment]
Requires https://github.com/ipa320/cob_common/pull/195 and https://github.com/ipa320/cob_calibration_data/pull/105

**Comment by NadiaHG** [comment]
@ipa-fmw can be merged? travis is not able to compile the current cob_robots (indigo_dev) version, this PR fix the problem

**Comment by fmessmer** [comment]
Just wondering why the cam is called head_cam after the mount position rather than using the manufacturer/model/camera_type as the name for the makro...

**Comment by NadiaHG** [comment]
@ipa-fxm you mean the name of the .urdf.xacro file on cob_description?  it is a generic web cam, actually the driver used is also a generic usb driver https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/drivers/usb_camera_node.launch. But I have no problems to rename the files...

File: cob_bringup/robots/cob4-2.xml [hunk 1]
```diff
@@ -305,7 +306,7 @@
         <include file="$(find cob_bringup)/drivers/image_flip.launch">
             <arg name="robot" value="$(arg robot)"/>
             <arg name="camera_name" value="head_cam"/>
-            <arg name="colorimage_in" value="/head_cam/image_raw"/>
+            <arg name="colorimage_in" value="/head_cam/image_rect_color"/>
             <arg name="colorimage_out" value="/head_cam_upright/image_raw"/>
```
Comment (floweisshardt) [inline cob_bringup/robots/cob4-2.xml:310]: I'll rename that to `head_cam_upright/image_rect_color` as soon as cob4-1 is back next week. image_rect_color is what the topic delivers, no more image_raw.

File: cob_hardware_config/cob4-5/config/image_flip_params_head_cam.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# rotation mode (0=fixed angle as defined  by rotation_angle, 1=automatic upright rotation against gravity using reference coordinate frame with upwards directed z-axis, 2=as 1 but image rotation only in 90deg steps -> faster)
+# int
+rotation_mode: 0
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-5/config/image_flip_params_head_cam.yaml:3]: I'd like to test rotation_mode=1 as soon as cob4-1 is back next week. (rotation_mode=1 should work now with modifications to image_flip by @ipa-rmb https://github.com/ipa320/cob_perception_common/pull/65).

**Comment by floweisshardt** [comment]
I wil cherry-pick 9af598d0b5d1eb3c23cd5263c4b8f0fd06065063 and open a new PR to satisfy Travis until the above issues are resolved -->https://github.com/ipa320/cob_robots/pull/543

**Review by floweisshardt — APPROVED** [review]
link is ok, but I couldn't get image flip to work with mode=1, we'll track this in a separate issue

**Pull request merged**